wiki/Beatmap/Pattern/osu!/Jump/en.md

```diff
@@ -2,10 +2,17 @@
 stub: true
 tags:
   - jumps
+  - cross screen jumps
 ---
 
 # Jump
 
 A **jump** is an [osu!](/wiki/Game_mode/osu!) [pattern](/wiki/Beatmap/Pattern) where two [hit objects](/wiki/Gameplay/Hit_object) are spaced far apart on the playfield compared to the spacing between other objects. Jumps usually signal high intensity in a song and they are most commonly snapped to 1/2 ticks on the [timeline](/wiki/Client/Beatmap_editor/Timelines#hit-objects).
 
 Jumps are usually played with snappy movement, as opposed to the smooth movement of [streams](/wiki/Beatmap/Pattern/osu!/Stream).
+
+# Cross Screen Jumps
```

Comment:
If this section is considered as a sub-section for `Jump`, more than one `#` is the suitable syntax(btw number of `#`s represent level, max 6).